lib/rack/cache/meta_store.rb

```diff
@@ -34,11 +34,11 @@ def lookup(request, entity_store)
 
       # find a cached entry that matches the request.
       env = request.env
-      match = entries.detect{ |req,res| requests_match?((res['Vary'] || res['vary']), env, req) }
+      match = entries.detect{ |req,res| requests_match?((res['vary'] || res['vary']), env, req) }
```

Comment:
Was it intentional to check the same value twice? Before, it was checking if there was a match for both the Capitalized `Vary` and the lowercase `vary`. But now that everything is lowercase, we only need to check one of them, right?

Reply:
Yes, that's correct, do you have time to submit a PR fixing this? Thanks!